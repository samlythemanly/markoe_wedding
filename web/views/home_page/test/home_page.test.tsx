import { cleanup, render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';

import { HomePage } from '..';

describe(HomePage.name, () => {
  afterEach(cleanup);

  test('renders properly', () => {
    const page = render(
      <MemoryRouter>
        <HomePage />
      </MemoryRouter>,
    );

    expect(page).toMatchSnapshot();
  });
});
