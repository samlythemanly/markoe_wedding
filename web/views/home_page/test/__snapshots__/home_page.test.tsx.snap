// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders properly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiBox-root css-1h7vr4f"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column css-40brdg-MuiGrid-root"
        >
          Sam
          <h1
            class="MuiTypography-root MuiTypography-h1 css-1gnxqcb-MuiTypography-root"
          >
            &
          </h1>
          Stephanie
        </div>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-direction-xs-column css-2gnp5z-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          >
            June 3rd, 2023 5:00PM
          </div>
          <div
            class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          >
            Malibou Lake Lodge, Agoura Hills, CA
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-spacing-xs-2 css-1mqp8pp-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item css-15zuw48-MuiGrid-root"
            >
              <a
                class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-ulkk9k-MuiTypography-root-MuiLink-root"
                href="/rsvp"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeLarge MuiButton-textSizeLarge MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeLarge MuiButton-textSizeLarge css-kpm0tx-MuiButtonBase-root-MuiButton-root"
                  tabindex="0"
                  type="button"
                >
                  <h6
                    class="MuiTypography-root MuiTypography-h6 css-tjvwag-MuiTypography-root"
                  >
                    RSVP
                  </h6>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </a>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item css-15zuw48-MuiGrid-root"
            >
              <a
                class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-ulkk9k-MuiTypography-root-MuiLink-root"
                href="/registry"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeLarge MuiButton-textSizeLarge MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeLarge MuiButton-textSizeLarge css-kpm0tx-MuiButtonBase-root-MuiButton-root"
                  tabindex="0"
                  type="button"
                >
                  <h6
                    class="MuiTypography-root MuiTypography-h6 css-tjvwag-MuiTypography-root"
                  >
                    Registry
                  </h6>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </a>
            </div>
          </div>
        </div>
        <div
          class="MuiBox-root css-k7ubg3"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiBox-root css-1h7vr4f"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column css-40brdg-MuiGrid-root"
      >
        Sam
        <h1
          class="MuiTypography-root MuiTypography-h1 css-1gnxqcb-MuiTypography-root"
        >
          &
        </h1>
        Stephanie
      </div>
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-direction-xs-column css-2gnp5z-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
        >
          June 3rd, 2023 5:00PM
        </div>
        <div
          class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
        >
          Malibou Lake Lodge, Agoura Hills, CA
        </div>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-spacing-xs-2 css-1mqp8pp-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-item css-15zuw48-MuiGrid-root"
          >
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-ulkk9k-MuiTypography-root-MuiLink-root"
              href="/rsvp"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeLarge MuiButton-textSizeLarge MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeLarge MuiButton-textSizeLarge css-kpm0tx-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                <h6
                  class="MuiTypography-root MuiTypography-h6 css-tjvwag-MuiTypography-root"
                >
                  RSVP
                </h6>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </a>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item css-15zuw48-MuiGrid-root"
          >
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-ulkk9k-MuiTypography-root-MuiLink-root"
              href="/registry"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeLarge MuiButton-textSizeLarge MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeLarge MuiButton-textSizeLarge css-kpm0tx-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                <h6
                  class="MuiTypography-root MuiTypography-h6 css-tjvwag-MuiTypography-root"
                >
                  Registry
                </h6>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </a>
          </div>
        </div>
      </div>
      <div
        class="MuiBox-root css-k7ubg3"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
